packages/components/src/range-control/tooltip.js

```diff
@@ -49,17 +49,17 @@ export default function SimpleTooltip( {
 }
 
 function useTooltipPosition( { inputRef, position: positionProp } ) {
-	const [ position, setPosition ] = useState( 'top' );
+	const [ position, setPosition ] = useState( 'bottom' );
 
 	const calculatePosition = useCallback( () => {
 		if ( inputRef && inputRef.current ) {
 			let nextPosition = positionProp;
 
 			if ( positionProp === 'auto' ) {
-				const { top } = inputRef.current.getBoundingClientRect();
-				const isOffscreenTop = top - TOOLTIP_OFFSET_HEIGHT < 0;
+				const { bottom } = inputRef.current.getBoundingClientRect();
+				const isOffscreenBottom = bottom - TOOLTIP_OFFSET_HEIGHT < 0;
 
-				nextPosition = isOffscreenTop ? 'bottom' : 'top';
+				nextPosition = isOffscreenBottom ? 'top' : 'bottom';
```

Comment:
Similar to the previous comment, I'm not sure this is needed if `position` is added as a prop to `SimpleTooltip`. It looks like this hook bypases all the logic in the inner `if` statement if `positionProp` has a value other than `auto`.

Reply:
Do we really need an `auto` position? I don't see the need there. The only positions making sense to me are `bottom` and `top`. I stand corrected though.